Makefile

```diff
@@ -0,0 +1,76 @@
+CHS_ENV_HOME  ?= $(HOME)/.chs_env
+TESTS         ?= ./...
+
+bin           := payment-reconciliation-consumer
+test_path     := ./test
+chs_envs      := $(CHS_ENV_HOME)/global_env $(CHS_ENV_HOME)/payment-reconciliation-consumer/env
+source_env    := for chs_env in $(chs_envs); do test -f $$chs_env && . $$chs_env; done
+xunit_output  := test.xml
+lint_output   := lint.txt
+
+commit        := $(shell git rev-parse --short HEAD)
+tag           := $(shell git tag -l 'v*-rc*' --points-at HEAD)
+version       := $(shell if [[ -n "$(tag)" ]]; then echo $(tag) | sed 's/^v//'; else echo $(commit); fi)
+
+.PHONY: all
+all: build
+
+.PHONY: fmt
+fmt:
+	go fmt ./...
+
+.PHONY: deps
+deps:
+	go get github.com/mitchellh/gox
```

Comment:
This sections should just `go get ./...` - we shouldn't specify specific dependencies if we can avoid